WooCommerce/WooCommerceTests/Extensions/StringWooTests.swift

```diff
@@ -21,4 +21,25 @@ final class StringWooTests: XCTestCase {
 
         XCTAssertEqual(sanitizedURL, expectedResult)
     }
+
+    func testSchemeIsTrimmedForURLWithoutPath() {
+        let expectedResult = "automattic.com"
+        let url = "https://automattic.com"
+
+        XCTAssertEqual(url.trimHTTPScheme(), expectedResult)
+    }
+
+    func testSchemeIsTrimmedForURLWithPath() {
+        let expectedResult = "automattic.com/work-with-us"
+        let url = "https://automattic.com/work-with-us"
+
+        XCTAssertEqual(url.trimHTTPScheme(), expectedResult)
+    }
+
+    func testStringIsNotTrimmerIfItIsNotAURL() {
+        let expectedResult = "regular string"
+        let notAURL = expectedResult
+
+        XCTAssertEqual(notAURL.trimHTTPScheme(), expectedResult)
+    }
```

Comment:
Nice that you added the tests for `trimHTTPScheme()` 💯 

Nit: can you rename the tests following our [new convention](https://github.com/woocommerce/woocommerce-ios/blob/develop/docs/naming-conventions.md#test-methods)?

Reply:
Right! Thanks for pointing that out. I considered it, but decided to follow the convention that the rest of this particular XCTestCase subclass was following. 

I'll update!